src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const DAY_RENT = 40;
+  const LONG_TERM = 7;
+  const BASE_RENT = DAY_RENT * days;
+  const LONG_TERM_DISCOUNT = 50;
+  const MIDL_TERM = 3;
+  const MIDL_TERM_DISCOUNT = 20;
+
+  if (days >= LONG_TERM) {
+    return BASE_RENT - LONG_TERM_DISCOUNT;
+  }
+
+  if (days >= MIDL_TERM) {
+    return BASE_RENT - MIDL_TERM_DISCOUNT;
```

Comment:
There is a logical error in the discount application. If `days` is greater than or equal to `LONG_TERM`, the function returns immediately with the `LONG_TERM_DISCOUNT`, without checking if the `MIDL_TERM_DISCOUNT` should also be applied. Consider revising the logic to ensure that the correct discount is applied based on the number of days.